src/app/confirmation-dialog/confirmation-dialog.component.html

```diff
@@ -1,11 +1,11 @@
-<h1 *ngIf="data?.title" mat-dialog-title>{{ data.title }}</h1>
+<h1 *ngIf="data?.title" mat-dialog-title data-cy="confirmation-dialog-title">{{ data.title }}</h1>
 
 <div *ngIf="data?.message" mat-dialog-content [innerHTML]="data.message"></div>
-<div mat-dialog-actions fxLayout="row" fxLayoutAlign="end center" fxLayoutGap="10px">
-  <button data-cy="cancel-dialog-button" mat-flat-button [mat-dialog-close]="false" class="secondary-1" cdkFocusInitial>
+<mat-dialog-actions [align]="'end'">
```

Comment:
I changed the uses of fxLayout in dialogs like this line to use the existing `mat-dialog-actions` align property to align it to the end of the row. It achieves the same effect without requiring the flex module.